{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--c16353b4-87a1-47af-a20b-d726429bd9c3",
    "created": "2020-05-26T20:30:04.749005Z",
    "modified": "2020-05-26T20:30:04.749005Z",
    "name": "CVE-2019-20428",
    "description": "In the Lustre file system before 2.12.3, the ptlrpc module has an out-of-bounds read and panic due to the lack of validation for specific fields of packets sent by a client. The ldl_request_cancel function mishandles a large lock_count parameter."
}